### PR TITLE
Bookstore:Styling

### PR DESCRIPTION
**Pull request by SelmaNdapanda**
## In this pull request, I implemented the following:

- I used the information detailed in Zeplin to style the Bookstore website.

### CODE REVIEW

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @SelmaNdapanda , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by SelmaNdapanda** [comment]
Okay, I'm sorry. I did not notice the linter error. I will sort it right away. Thank you

File: src/components/Navbar.js [hunk 1]
```diff
@@ -5,14 +5,14 @@ const Navbar = () => (
   <header className="navbar">
```
Comment (Nemwel-Boniface) [inline src/components/Navbar.js:6]: - [x] Kindly recheck the styling for this section. There is a right icon that is supposed to be there according to the design in Zeplin but is not in yours.

**This is how I am seeing your design:**

image

**This is the desired design:**

image

File: src/components/Addbook.js [hunk 2]
```diff
@@ -26,7 +26,8 @@ const Addbook = () => {
   };
 
   return (
-    <div>
+    <div className="addCard">
```
Comment (Nemwel-Boniface) [inline src/components/Addbook.js:29]: - [x] Kindly recheck the styling for this section. I am actually not able to see the add book button which is supposed to be present according to the Zeplin design.

**This is how I am seeing your design:**

image

**This is the desired design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @SelmaNdapanda     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**